app/src/main/java/com/teamagam/gimelgimel/app/map/actions/timeplay/TimeplayViewModel.java

```diff
@@ -200,4 +252,61 @@ public void clearMap() {
       mDisplayed.clear();
     }
   }
+
+  public class TextTimeDisplayer implements RadialTimePickerDialogFragment.OnTimeSetListener {
+
+    private boolean mIsInEditOfStartDate;
+    private Calendar mResultCalender;
+
+    public TextTimeDisplayer(boolean isInEditOfStartDate) {
```

Comment:
isInEditOfStartDate ?!@#?!@#?@!#

Reply:
Ask for a callback that updates end/start/whatever instead of passing this boolean.
It read bad when called and definitely non-standard or clean

Reply:
or herniation.

Reply:
done